Dockerfile.php8.0

```diff
@@ -0,0 +1,50 @@
+FROM public.ecr.aws/docker/library/alpine:3.16
+
+ARG WP_VERSION=6.6
```

Comment:
Same for all docker files, let's not have this be a default value, make it required